tests/core/test_scheduler.py

```diff
@@ -16,9 +17,11 @@
                     schedule_and_update_computed_tokens)
 
 
-def test_scheduler_add_seq_group():
+@pytest.mark.parametrize('use_v2_block_manager', [True, False])
```

Comment:
Since we are deprecating block manager v1, we could only test block manager v2.

Reply:
These tests run quite fast. So I was wondering if we can retain both till block manager v1 is removed. I am suggesting keeping it so that we continue to have test coverage till it is removed. Let me know.

Reply:
That's also a fair call. Ok let's keep both them.